docs/CodingGuidelines.md

```diff
@@ -1,201 +1,217 @@
-
-# Coding guidelines
+# Coding Guidelines
```

Comment:
should use sentence casing for the headlines

Reply:
Updated.